driver/src/main/scala/core/nodeset.scala

```diff
@@ -663,16 +667,20 @@ final class ChannelFactory private[reactivemongo] (
     pipeline
   }
 
-  private def sslContext = {
+  private def sslContext(tm: Array[javax.net.ssl.TrustManager]) = {
+    import java.io.FileInputStream
+    import java.security.{ KeyStore, Security }
+    import javax.net.ssl.KeyManagerFactory
 
-    val keyManagers: Array[KeyManager] = Option(System.getProperty("javax.net.ssl.keyStore")).map { path =>
+    val keyManagers = Option(System.getProperty("javax.net.ssl.keyStore")).map { path =>
 
       val password = Option(System.getProperty("javax.net.ssl.keyStorePassword")).getOrElse("")
```

Comment:
I'm using that orNull now a bit further down inside the call to init, must of pushed that just after you commented here